ext/node/crypto/cipher.rs

```diff
@@ -156,6 +173,12 @@ impl Decipher {
           decryptor.decrypt_block_b2b_mut(input.into(), output.into());
         }
       }
+      Aes128Ecb(decryptor) => {
+        assert!(input.len() % 16 == 0);
```

Comment:
Is it possible to hit this assertions from userland?

Reply:
If the user directly called `ops.op_node_cipheriv_encrypt` with arbitrary length input, then this assert can fail. If the user use cipher only via Node.js compat APIs, this shouldn't fail.